code/game/atoms.dm

```diff
@@ -168,8 +171,19 @@ directive is properly returned.
 	if(loc)
 		return loc.return_gas()
 
+///returns if we can melt an object, but also the speed at which it happens. 1 just means we melt it. 0,5 means we need a higher strength acid. higher than 1 just makes it melt faster
+/atom/proc/dissolvability(acid_strength)
+	return 1
 
+//returns how long it takes to apply acid on this atom
+/atom/proc/get_acid_delay()
+	return 1 SECONDS
 
+///returns if we are able to apply acid to the atom, also checks if there is already a stronger acid on this atom
+/atom/proc/should_apply_acid(obj/effect/xenomorph/acid/new_acid)
```

Comment:
Proc is doing the opposite of what it says.

return true if you shouldn't apply acid

Reply:
Just invert the logic (and where we call the proc)